src/api/worker/user/use_case/login.py

```diff
@@ -0,0 +1,47 @@
+import re
+
+from datetime import datetime, timedelta
+from hashlib import sha512
+
+from src.api.worker.auth.models.credential_dto import CredentialDTO
+from src.api.worker.auth.models.token_dto import TokenDTO
+from src.api.worker.auth.use_case.token import Token
+from src.api.worker.user.exceptions import *
+from src.database.repositories import UserRepositoryInterface
+from src.database.repositories.token_repository import TokenRepositoryInterface
+
+
+class Login:
+
+    user_repository: UserRepositoryInterface
+    token_repository: TokenRepositoryInterface
+
+    def __init__(self,
+                 user_repository: UserRepositoryInterface,
+                 token_repository: TokenRepositoryInterface):
+        self.user_repository = user_repository
+        self.token_repository = token_repository
+
+    def worker(self, credential: CredentialDTO) -> TokenDTO:
```

Comment:
```suggestion
    def worker(self, credential: CredentialDTO) -> TokenDTO:
        """
        Documentez la fonction
        """
```
Faut que je le fasse aussi pour ma PR